Distraction Shield/modules/urlFormatter.js

```diff
@@ -179,7 +179,7 @@ function readyStateChange(xmlHttp, onSuccess, onFailure, theUrlToGet) {
             let title = titleTags[1];
             onSuccess(xmlHttp.responseURL, title);
         } else {
-            onSuccess(xmlHttp.responseURL, theUrlToGet);
+            onSuccess(xmlHttp.responseURL, 'theUrlToGet');
```

Comment:
ESLint couldn't find this variable, so I quoted it to see whether Travis would actually pass and whether the application would still work. Both worked. However, as I can see now, after pulling development the variable **is** actually defined, I just committed a fix. It was probably because  merging went wrong somewhere.